### ISSUE TITLE
Saving of Shops

### ISSUE THREAD

**Issue by Tommsy64** [moomaxie/BetterShops#16]
How come all the `saveConfig();`s are **commented out** in the FileShop??

The shops only when the plugin is disabled. In the event of an unexpected server shutdown, all the changes to shops would be **lost**!

**Comment by anemul** [comment]
Yes . That is exactly what happened to me last night . Our power went down unexpectedly and lost all of the items in the shop and the Villagers became regular villagers , other than this , the shop has been awesome .

**Comment by sameer** [comment]
@anemul @Tommsy64 Please see https://github.com/Robotia/BetterShops/releases
